### PR TITLE
Removed some content didn't think it was useful

### PR DESCRIPTION
**Pull request by forceedge01**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,40 +18,10 @@
 
 	<!-- Start Landing header-->
 	<div class="landing">
-		<h1>Digital Hybrids</h1>
-		<h4>Born To Conquer</h4>
+		<h1>Digital Hybrids Yo this is not true.</h1>
```
Comment (forceedge01) [inline index.html:21]: Dont like this title

File: index.html [hunk 2]
```diff
@@ -18,40 +18,10 @@
...
 
 		<div class="row home justify-content-center text-center px-md-5 px-lg-0">
 
-
```
Comment (Jawad63) [inline index.html:26]: leave it as it is man

Comment (forceedge01) [reply index.html:26]: Ok fine.

File: index.html [hunk 3]
```diff
@@ -18,8 +18,7 @@
 
 	<!-- Start Landing header-->
 	<div class="landing">
-		<h1>Digital Hybrids</h1>
-		<h4>Born To Conquer</h4>
+		<h1>Digital Hybrids Yo this is not true.</h1>
```
Comment (Jawad63) [inline index.html:21]: this is fine

**Pull request merged**